src/patterns/gen-ai/aws-contentgen-appsync-lambda/index.ts

```diff
@@ -486,7 +486,7 @@ export class ContentGenerationAppSyncLambda extends Construct {
         tracing: lambda_tracing,
         vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_WITH_EGRESS },
         securityGroups: [this.securityGroup],
-        memorySize: 1_769 * 1,
+        memorySize: 1_769 * 4,
```

Comment:
```suggestion
memorySize: lambdaMemorySizeLimiter(this, 1_769 * 4),
```

Reply:
if used in a workshop

Reply:
done - 2c2773e876d1a6fc4e48aacaf0b0e8e262826049